src/main/java/Dictionary/WordBank.java

```diff
@@ -0,0 +1,119 @@
+package Dictionary;
+
+import java.util.ArrayList;
+
+import exception.NoWordFoundException;
+import command.OxfordCall;
+import storage.Storage;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class WordBank {
+    private TreeMap<String, Word> wordBank;
+
+    public WordBank(Storage storage) {
+        wordBank = storage.loadFile();
+    }
+
+    public WordBank(TreeMap<String, Word> wordBank) {
+        this.wordBank = wordBank;
+    }
+
+    public TreeMap<String, Word> getWordBank() {
+        return wordBank;
+    }
+
+    public Word getWordAndMeaning(String word) throws NoWordFoundException {
+        if (!wordBank.containsKey(word)) {
+            throw new NoWordFoundException(word);
+        }
+        return wordBank.get(word);
+    }
+
+    public void addWord(Word word) {
+        this.wordBank.put(word.getWord(), word);
+    }
+
+    /**
+     * Looks up for meaning of a specific word
```

Comment:
Good to add here that "if word is not in Wordbank, call OxfordCall to find the word"